### ISSUE TITLE
Miscellaneous webform items

### ISSUE THREAD

**Issue by joycarpediem** [akvo/isco#79]
- [x] Show the "other" option when selected in the survey editor for option questions. A text box should be displayed on selecting "Other" . This is also the case for multi-select
- [x] Add an "Add Comment" button to every question. On clicking it , a text box appears where users can enter a comment
- [x] Styling of the webform
- [x] Support submitting comment field value (add comment column to answer table)
- [x] The repeat group button text is configurable
- [x] Show hover text in tooltip
- [x] Hide language dropdown

**Comment by dedenbangkit** [comment]
The `allowOther` option is available in [Semarapura v1.3.8](https://github.com/akvo/akvo-react-form/releases/tag/v1.3.8) along with `allowOtherText`.

For the comment questions, I think we should have another column in the question table, and remap the question group with the additional comment question after the actual question with dependency on the question itself.

Example:
```js
const questions = [
  { id: 1, name: 'Deden', allowComment: true },
  { id: 2, name: 'Galih' }
]

const result = questions.reduce((previous, current) => {
  let extended = [current]
  if (current?.allowComment) {
    extended = [
      current,
      {
        id: `C-${current.id}`,
        name: 'Deden Comment',
        dependency: current.id
      }
    ]
  }
  return [...previous, ...extended]
}, [])
```

**Comment by wayangalihpratama** [comment]
- [ ] Support submitting comment field value (add comment column to answer table)
